### DISCUSSION TITLE
Icon Request: icon-ebook (generic)

### DISCUSSION THREAD

**Discussion by saltedlolly** [FortAwesome/Font-Awesome#21145] [Icon Requests]
I'm a big fan of font awesome, but recently I found a use case it couldn't help me with.

I would really like to see a set of icons created for different types of books. This way when when creating links for a book from a website, it gives people an easy way to highlight the different editions available.

Examples:
- Hardback 
- Paperback 
- eBook (Amazon Kindle)
- eBook (Kobo)
- eBook (Nook)
- eBook (Google)
- eBook (Sony BBeB)
- eBook (Apple iBooks)
- eBook (ePub) 
  [URL: https://upload.wikimedia.org/wikipedia/en/thumb/1/14/EPUB_logo.svg/150px-EPUB_logo.svg.png]
- eBook (PDF)
- eBook (Generic)

**Comment by gquipster** [comment]
+1

**Comment by olololox** [comment]
+1

**Comment by Casara** [comment]
+1

**Comment by QWp6t** [comment]
:+1:

**Comment by tagliala** [comment]
This request does not meet our contributing guidelines anymore: https://github.com/FortAwesome/Font-Awesome/blob/master/CONTRIBUTING.md#requesting-new-icons

> Please do not request comprehensive sets of icons related to a very specific field.

I'm leaving this open for tracking purposes, but I should advise you to consider a custom build if you need  those icons in a short while, sorry!

**Comment by fabiograsso** [comment]
+1

**Comment by maggie-hasegawa** [comment]
+1

**Comment by olya-gary** [comment]
+1

**Comment by olya-gary** [comment]
just read the @tagliala comments ("Please do not request comprehensive sets of icons related to a very specific field.") Wondering why? - not enough resources to cover everything?

However, books, in my opinion, is not a "very specific field" as they are everywhere. I'm in the medical library and we would love to have those icons so our users - in medicine can easier recognize what's available. Would you please reconsider the 'book' icons?
thank you

**Comment by tagliala** [comment]
@olya-gary 

comprehensive requests like this one are hard to implement manage.

Given that a generic web application would benefit of an ebook icon, adding 11 icons only to represent e-books could be considered beyond the scope of this project

The best solution is a custom build: https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome

btw some popular requests like #492 and #2148 were included in the set.

**Comment by gquipster** [comment]
From the perspective of those of us in the printing/publishing industry these would be EXTREMELY useful.

**Comment by da-benny** [comment]
+1

**Comment by RichardAlbary** [comment]
+1

**Comment by DeNelo** [comment]
+1

**Comment by koimad** [comment]
+1

**Comment by fmorato** [comment]
+1

**Comment by brianmcmichael** [comment]
+1

**Comment by jenstroeger** [comment]
I am not sure about a generic “ebook” icon, but the [above list](https://github.com/FortAwesome/Font-Awesome/issues/2356#issue-22175676) of devices and their supported file formats may need some clarification.

I think that “ebook” is often depicted by the “EPUB file format” icon, i.e. this one: 

<img src="https://upload.wikimedia.org/wikipedia/en/thumb/1/14/EPUB_logo.svg/160px-EPUB_logo.svg.png"> ([source](https://en.wikipedia.org/wiki/EPUB))

but without the `ePUB` below the logo. However, this would also warrant renaming the icon to `icon-epub` considering that there are other “ebook” file formats around (e.g. [`.mobi`](https://en.wikipedia.org/wiki/Mobipocket) for Kindle, [etc.](https://en.wikipedia.org/wiki/Comparison_of_e-book_formats) although I would not consider `.pdf` an ebook file format).

Also, as a comment on issue https://github.com/FortAwesome/Font-Awesome/issues/7095: the icons requested in that issue are about ebook reading apps/devices which is different than ebook file formats. These app/device related icons are addressed by issues https://github.com/FortAwesome/Font-Awesome/issues/14127 & https://github.com/FortAwesome/Font-Awesome/issues/15290 (Nook, supports EPUB); https://github.com/FortAwesome/Font-Awesome/issues/13659 (Kobo, supports EPUB); or https://github.com/FortAwesome/Font-Awesome/issues/2113 and https://github.com/FortAwesome/Font-Awesome/issues/3288 (Kindle, supports MOBI and compatibles).

@tagliala, any thoughts on the timeline here? This particular request #2356 seems a bit unclear but it would be great to see progress on the ebook file formats icons (i.e. EPUB), as well the ebook reading devices (e.g. Nook, Kobo, Kindle, …).

**Comment by HannayAndy** [comment]
+1

**Comment by JacobDB** [comment]
I'd also like to see this, a generic icon kind of like this one:

![image](https://user-images.githubusercontent.com/1123997/166154241-dfea3416-4fc3-4421-9386-ac62515a2ff6.png)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
